pkg/export_format/export_format.go

```
@@ -0,0 +1,28 @@
+package export_format
+
+import (
+	"encoding/json"
+	"os"
+)
+
+type (
+	platform_version string
```

golangci-lint

type `platform_version` is unused

(unused)